### PR TITLE
Changes to MRI resources

### PR DESCRIPTION
**Pull request by Micahh222**

### CODE REVIEW

File: mri.md [hunk 1]
```diff
@@ -1,36 +1,53 @@
...
+  - [Youtube (open-source)][p_fmri_youtube]
+- MRI physics
+  - [Assaf Tal Lectures][physics_mri]
+- Machine learning in Neuroscience
+  - [Neurohackademy Lectures][neurohack]
+- Introduction to PCA
+  - [PCA][intro_PCA]
 
-## Books that should be available
+## Books available by hardback or pdf
```
Comment (handwerkerd) [inline mri.md:47]:
```suggestion
## Books (ask lab members for copies)
```

File: mri.md [hunk 2]
```diff
@@ -51,3 +68,8 @@
 [afni_handouts]: https://afni.nimh.nih.gov/class_handouts
 [pete_molfese]: https://blog.cogneurostats.com
 [layerfmri]: https://layerfmri.com
+[huettel_pdf]: https://github.com/Micahh222/lab-docs/files/6970668/fMRI_Huettel.pdf
```
Comment (handwerkerd) [inline mri.md:71]:
```suggestion
```

File: mri.md [hunk 3]
```diff
@@ -1,36 +1,53 @@
...
+- Machine learning in Neuroscience
+  - [Neurohackademy Lectures][neurohack]
+- Introduction to PCA
+  - [PCA][intro_PCA]
 
-## Books that should be available
+## Books available by hardback or pdf
 
-- Functional Magnetic Resonance Imaging by Huettel, Song, & McCarthy (designed for undergrad introductory courses)
+- [Functional Magnetic Resonance Imaging by Huettel, Song, & McCarthy (designed for undergrad introductory courses)][huettel_pdf]
```
Comment (handwerkerd) [inline mri.md:49]: We shouldn't be sharing pdfs of a copyrighted book on our lab website.

**Review by handwerkerd — CHANGES_REQUESTED** [review]
Good additions. Just remove the link to the Huettel pdf.

**Review by handwerkerd — APPROVED** [review]

**Comment by Micahh222** [comment]
@jbteves

**Review by jbteves — APPROVED** [review]

**Comment by jbteves** [comment]
One note: when opening a pull request, if you make an initial comment like
```
Closes #123
```
Then when the pull request is merged, that issue will automagically be closed.

**Comment by Micahh222** [comment]
Thank you Josh :)

**Pull request merged**